src/lib/address/utils.ts

```diff
@@ -0,0 +1 @@
+export const isEthAddress = (address: string): boolean => /^0x[0-9A-Fa-f]{40}$/.test(address)
```

Comment:
Just found out about https://docs.ethers.io/v5/api/utils/address/#utils-getAddress, what do you think about using it in a follow-up PR instead?